data/tests/versioning/test_versioner.py

```diff
@@ -0,0 +1,51 @@
+from os import path
+from pathlib import Path
+from data.versioning.versioner import WandbDatasetVersioner, VersioningConfig
```

Comment:
Do we need anything besides `WANDB_API_KEY` in CI env to make this work?

Reply:
No